content/en/v3/admin/platforms/google/svc_acct.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Configure Google Service Account
+linktitle: GCP Service Account
+type: docs
+weight: 100
+aliases:
+---
+
+This doc will demonstrate how to set up a Google service account that can be used by Terraform to execute [Jenkins X GKE Module](https://github.com/jenkins-x/terraform-google-jx#jenkins-x-gke-module)
+
+> ⚠️  This doc has been designed to assist in performing the demonstration through copying and pasting each block of code into a shell terminal.<br>
```

Comment:
same comments as before, the triangle and br tag ...

Reply:
I saw this '> ⚠️' notation on another page and I liked how separated certain content details. I attempt to use it for tips, suggestions, warnings, etc. I guess the '⚠️' does imply warning and perhaps I should use '> 💡'